README.md

```diff
@@ -9,28 +9,26 @@ This repository contains the sources of the OCaml website. It is served at https
 - **Integrated documentation and package management:** The site combines the
   package management (currently opam.ocaml.org) with a new central
   documentation source (codenamed 'docs.ocaml.org') for all 14000+ opam packages
-  directly within the ocaml.org site.
+  directly within the OCaml.org site.
 
 - **Responsive and accessible:** The site design also takes into account modern
   web-design principles, restructuring the old content in accordance with methods
   that will present it more compellingly. It is a total redesign that modernises
-  the look and feel of the webpage, as well as make it easier to navigate, and more
+  the look and feel of the webpage, as well as make it easier to navigate and more
   accessible (particularly on mobile devices).
 
 - **Separation of data editing from HTML/CSS generation:** The data used in the website
-  is stored in Yaml or Markdown so users can easily edit it and contribute to the website.
-  We generate OCaml code from this data to serve the site content and and Ocurrent to
+  is stored in YAML or Markdown, so users can easily edit it and contribute to the website.
+  We generate OCaml code from this data to serve the site content and OCurrent to
   automate the data pipelines. All the data used in the site can be found in [`./data`](./data).
```

Comment:
```suggestion
  We generate OCaml code from this data to serve the site content and for OCurrent to
  automate the data pipelines. All the data used in the site can be found in [`./data`](./data).
```
Does "we generate OCaml code" refer to both "serve the site content" and "OCurrent to automate" ?? Or is OCurrent automating separate from this code generation?

Reply:
I've rewritten this sentence